index.template.html

```diff
@@ -2992,6 +2992,73 @@ <h2>Identification</h2>
   CoAP Content-Format ID <code>432</code> (see <a href="#iana-section"></a>).
   </p>
 
+</section>
+
+<section id="validation-serialization-json">
+<h2>Validation</h2>
+
+  <p>
+  In several contexts automatic validation of a JSON-based serialization 
+  of a Thing Description is useful.  Formally, a valid TD must statisfy all
+  the assertions in this specification, but not all assertions can be 
+  validated given only the JSON serialization, for instance, the assertions
+  listed under <a>Behavioral Assertions</a> that relate a TD to the
+  behavior of a Thing that it describes.  Extensions are also problematic,
+  in that even if formal metadata is given for validating an extension,
+  dynamically fetching this metadata in a deployment might pose a privacy
+  risk.  In this section, therefore, we name and define various levels of
+  validation appropriate for different contexts.
+  </p>
+  <section id="basic-validation-serialization-json">
+  <h3>Basic Validation</h3>
+  <p>Basic Validation is appropriate where validation needs to be 
+  self-contained (e.g. devices on isolated networks).
+  It does not attempt to validate context extensions and vocabularies.
+  </p><p> 
+  In practice, these assertions can be validated using a JSON Schema 
+  in combination with a few spot checks, for example to check that 
+  security schema names have matching definitions.
+  </p><p> 
+  This level of validation includes all assertions implied by 
+  normative tables in this document as well as all assertions marked
+  with the "Basic" tag.
+  </p>
+  </section>
+  <section id="intermediate-validation-serialization-json">
+  <h3>Intermediate Validation</h3>
```

Comment:
I'm not sure if "Intermediate" is the best naming. If we use "Minimum Validation" above, maybe we can use "Basic Validation" here?